content/en/os/1.14.x/api/settings/pki/_index.markdown

```diff
@@ -0,0 +1,15 @@
++++
+title="pki"
+type="docs"
+toc_hide=true
+description="Settings related to Custom CA Certificates (`settings.pki.*`)"
+
++++
+
+By default, Bottlerocket ships with the [Mozilla CA certificate store](https://ccadb.my.salesforce-sites.com/mozilla/IncludedCACertificateReport), but you can add self-signed certificates with `settings.pki.<bundle name>`.
```

Comment:
I'd love feedback if the link "Mozilla CA certificate store" is appropriate.

Reply:
I would prefer either https://curl.se/docs/caextract.html or https://wiki.mozilla.org/CA/Included_Certificates, and let the users navigate to the page your showed. I don't like the advertisement on the `salesforce` website :sweat_smile:

Reply:
Alright, I'll swap it over to the mozilla link. FWIW, I don't think it has anything to do with salesforce, I think that Mozilla just uses salesforce. But you're right - if we are saying Mozilla, we should make it crystal clear this is from them and the link to salesforce's domain confuses that.